Tests/L0/SonarQubePostTest/PRCA/OrchestratorTests.ps1

```diff
@@ -95,3 +90,16 @@ Assert-AreEqual 5 ($comments | Where-Object {$_.line -eq 5}).Priority "Info issu
 
 Assert-AreEqual 6 ($comments | Where-Object {$_.line -eq 6}).Priority "No severity issue - priority 6"
 Assert-AreEqual 6 ($comments | Where-Object {$_.line -eq 7}).Priority "Other severity issue - priority 6"
+
+#
+# Test 4 - Line 
```

Comment:
A short description of the test would be useful e.g. "Missing and empty line numbers should be reported as zero"